IBuildaAZoo/IBuildaAZoo/Cow.cs

```diff
@@ -3,8 +3,13 @@ namespace IBuildaAZoo
 {
     public class Cow
     {
-        public Cow()
+        public abstract class Cow()
         {
-        }
+          // Properties
+        public abstract int NumberOfLegs = 2;
+        public string Size = "Large"
+        public virtual string Skin = "Fur";
```

Comment:
Once `Cow` inherits from `Animal`, `NumberOfLegs` and `Skin` need to `override`.
```suggestion
        public override int NumberOfLegs => 4;
        // public string Size = "Large" // This should be assigned in the constructor
        // public virtual string Skin = "Fur"; // Don't need to override because Fur is correct
```